cli/src/main.rs

```diff
@@ -62,6 +62,13 @@ fn run() -> Result<(), CliError> {
                 (@arg quiet: -q --quiet "do not display output")
             )
         )
+        (@subcommand health =>
+            (about: "Splinter commands for inspecting network integrety and health")
```

Comment:
Should match the starting case of the other descriptions - same for the `status` subcommand.